db/migrate/20220511185320_add_renewal_eligible_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddRenewalEligibleToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column(:users, :renewal_eligible, :boolean)
```

Comment:
Why is this field in Accounts?

Reply:
Accounts is already providing the users contact id from Salesforce and doing the heavy job of syncing with contacts - so instead of adding this to the CMS.. I thought it was best to add it here.

It should eventually be moved to the salesforce-api - along with the contact id, faculty status, etc.

Reply:
ah ok that makes sense.